.github/workflows/Auto-merge-dependabot-PRs.yml

```diff
@@ -0,0 +1,23 @@
+# This workflow triggers auto-merge of any PR that dependabot creates so that
+# PRs will be merged automatically without maintainer intervention if CI passes
+name: Auto-merge dependabot PRs
+
+on:
+  pull_request_target:
+    types: [opened]
+
+jobs:
+  auto-merge:
+    if: github.repository == 'sharkdp/bat' && startsWith(github.head_ref, 'dependabot/')
```

Comment:
So this is save because we also check the repo, i.e. it will not work for a fork?

Reply:
The safety comes more from the fact that GitHub will not give forks access to repo secrets. This is more here to avoid the annoyance of triggering a job that can't succeed because of a missing secret :)